common/lib/xmodule/xmodule/html_module.py

```diff
@@ -368,6 +368,13 @@ class StaticTabFields(object):
         scope=Scope.settings,
         default="Empty",
     )
+    course_staff_only = Boolean(
+        display_name=_("Hide Page From Learners"),
+        help=_("If this option is selected, only course staff will see this page"
```

Comment:
@dmitry-viskov Please update the help text to the string that @catong suggested above (include the period at the end). Then squash the change.

I have gotten the OK from @sstack22 to merge this PR without the doc being done.